packages/protocol/migrations/23_elect_validators.ts

```diff
@@ -201,6 +235,20 @@ module.exports = async (_deployer: any, networkName: string) => {
     artifacts
   )
 
+  if (networkName === 'development') {
```

Comment:
Can we avoid hard-coding like this? Maybe read the mnemonic from the truffle config?

Reply:
Probably, I'll try to check if there is some easy way.

Reply:
we can define a boolean key `ganache` in `migrationsConfig`; and then override it when calling migrate